build-logic/plugins/src/main/java/ch/srgssr/pillarbox/gradle/PillarboxAndroidApplicationPlugin.kt

```diff
@@ -49,7 +49,7 @@ class PillarboxAndroidApplicationPlugin : Plugin<Project> {
                 register("release") {
                     val password = System.getenv("DEMO_KEY_PASSWORD") ?: extra.properties["pillarbox.keystore.password"] as String?
 
-                    storeFile = file("./demo.keystore")
+                    storeFile = file("../config/keystore/demo.keystore")
```

Comment:
You can provide a path relative to the root project:
```suggestion
                    storeFile = rootProject.projectDir.resolve("config/keystore/demo.keystore")
```